src/main/resources/lang/en-GB.json

```diff
@@ -1119,7 +1119,10 @@
     "guild#flag#list": {
       "name": "list",
       "description": "Lists all flags."
-    }
+    },
+    "pos": "position",
+    "source": "source",
+    "destination": "destination"
```

Comment:
Just a finicky thing but could this be above the other arguments for the sake of consistency ❤️